PrivateTalkApp/Common/Repositorys/ResponseBodys/WorldTimeResponse.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  WorldTimeResponse.swift
+//  PrivateTalkApp
+//
+//  Created by 都甲裕希 on 2024/08/27.
+//
+
+import Foundation
+
+struct WorldTimeResponse: Decodable {
+    
+    // 現在の時刻（UTC）
+    let datetime: String?
+    
+    /// アプリ内で使える形式に直すために新しくインスタンス化する
+    func updatedDatetimeProperty(newDatetime: String?) -> WorldTimeResponse {
+        return WorldTimeResponse(datetime: newDatetime)
+    }
```

Comment:
こいつ要らなそうだし、もし必要だとしても.initにした方が良さそう。

```suggestion
    init(datetime: String?) {
        self.datetime = datetime
    }
```

Reply:
Fixed.